IT076/Inventory_Management/Inventory_Management/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System.Data;
+using System;
+using System.Data.SqlClient;
+using System.Net;
+using System.Reflection;
+using System.Xml.Linq;
+
+namespace InventoryManagementSystem
+{
+    class Program
+    {
+            static string connectionString = "Server=localhost\\SQLEXPRESS;Database=Inventory_Management;Trusted_Connection=True;TrustServerCertificate=True;";
+
+        static void Main(string[] args)
+        {
+
+            while (true)
+            {
+                Console.WriteLine("Inventory Management System");
+                Console.WriteLine("1. Add Item");
+                Console.WriteLine("2. Update Quantity");
+                Console.WriteLine("3. View Inventory");
+                Console.WriteLine("4. Delete Item");
+                Console.WriteLine("5. Exit");
+                Console.Write("Enter your choice: ");
+                int choice = Convert.ToInt32(Console.ReadLine());
+
+                switch (choice)
+                {
+                    case 1:
+                        AddItem();
+                        break;
+                    case 2:
+                        UpdateQuantity();
+                        break;
+                    case 3:
+                        ViewInventory();
+                        break;
+                    case 4:
+                        DeleteQuantity();
+                        break;
+                    case 5:
+                        Environment.Exit(0);
+                        break;
+                    default:
+                        Console.WriteLine("Invalid choice.");
+                        break;
+                }
+            }
+        }
+
+        static void AddItem()
+        {
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                Console.Write("Enter item name: ");
+                string itemName = Console.ReadLine();
+                Console.Write("Enter quantity: ");
+                string squantity = Console.ReadLine();
+                int quantity = int.Parse(squantity);
+
+                try
+                {
+
+                    using (SqlCommand cmd = new SqlCommand("AddItemProcedure", connection))
+                    {
+                        cmd.CommandType = CommandType.StoredProcedure;
+
+                        cmd.Parameters.AddWithValue("@ItemName", itemName);
+                        cmd.Parameters.AddWithValue("@Quantity", quantity);
+
+                        cmd.ExecuteNonQuery();
+
+                        Console.WriteLine("Item information inserted successfully!\n");
+                    }
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("An exception occurred " + e);
```

Comment:
its better to always add e.stacktrace and e.message to know the exception details precisely